OpenRA.Mods.Common/Widgets/Logic/MissionBrowserLogic.cs

```diff
@@ -181,7 +191,7 @@ protected override void Dispose(bool disposing)
 
 		void CreateMissionGroup(string title, IEnumerable<MapPreview> previews, Action onExit)
 		{
-			var header = ScrollItemWidget.Setup(headerTemplate, () => true, () => { });
+			var header = ScrollItemWidget.Setup(headerTemplate, () => false, () => { });
```

Comment:
This is a hack to make the headers have a different appearance:
![image](https://user-images.githubusercontent.com/1355810/184932058-c7463bed-cdc6-4f83-b4d8-e38054f62da3.png)
Without it they look wrong:
![image](https://user-images.githubusercontent.com/1355810/184932230-949c3e71-43ca-4b75-9115-6ede168cc9f6.png)

I guess the proper fix here would be to change the background of the ScrollItem in the header template. But in any case it's not related to what this PR is doing so please revert that line.

Reply:
It is related, as when you click the play button the Map in mission selector needs to be preselected. I'll adjust the visuals

Reply:
I see. But then again - do we want to preselect the last edited mission? This doesn't seem like the right UX to me.

Reply:
If you edit the mission you are very likely going to want to play it. I've applied the same reasoning as I did with MapChooser